datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -120,6 +151,56 @@ use super::AggregateExec;
 /// hash table).
 ///
 /// [`group_values`]: Self::group_values
+///
+/// # Spilling
+///
+/// The sizes of group values and accumulators can become large. Before that causes out of memory,
+/// this hash aggregator outputs those data early for partial aggregation or spills to local disk
```

Comment:
```suggestion
/// for partial aggregation, this hash aggregator emits partial states early. For final aggregation, it spills input partial state into local disk.
```

Reply:
I took some of your suggestion and revised

Reply:
Thank you.